### PR TITLE
Fix #140&#147

### LINKED ISSUE #140: Change start screen title and remove menu duplication
from EasyGrade to RubricApp

also navigation menu code should be moved to parent.html (or loggedin.html?), which extends bootstrap/base.html

signup/login page duplication too, different parent than parent.html (e.g., loggedout.html?)

### LINKED ISSUE #142: Remove "some message" from left panel
image

On the evaluation page, remove the "some message"

### LINKED ISSUE #147: Identifying other html template duplications
comes after #140 fixes

### PR DESCRIPTION
**Pull request by zhouhuan200010pm**
Fixes #140 
Fixes #142 
Fixes #147

### CODE REVIEW

File: templates/project.html [hunk 1]
```diff
@@ -149,7 +146,7 @@
     <label class="nav-label" for="toggle">&#9776</label>
     <input type="checkbox" id="toggle"/>
     <div class="menu">
-        <span class="Logo"><a href="#" >EasyGrade</a></span>
+        <span class="Logo"><a href="#" >OpenRubric</a></span>
```
Comment (bmyerz) [inline templates/project.html:149]: Is there a good reason why project.html doesn't use the nav bar from parent_signed_in?

File: templates/project_profile_backed_up.html [hunk 2]
```diff
@@ -1,8 +1,4 @@
-{% extends "bootstrap/base.html" %}
```
Comment (bmyerz) [inline templates/project_profile_backed_up.html:1]: this template doesn't seem to be used by app.py. You should delete it as part of this pull request.

File: templates/create_project.html [hunk 3]
```diff
@@ -37,7 +34,7 @@
     width: 80%;
     padding-left: 20px;
 }
-.nav{
+.nav-body{
```
Comment (bmyerz) [inline templates/create_project.html:37]: During the weekly meeting you discussed why you are leaving this in the child files, and that is okay. To make sure we do not forget in the future, can you put comments above these .nav-body styles that explain why you chose not to move the .nav-body style to the parent_signed_in file?

File: templates/instructor_dashboard.html [hunk 4]
```diff
@@ -41,7 +37,7 @@
     width: 80%;
     padding-left: 20px;
 }
-.nav{
+.nav-body{
```
Comment (bmyerz) [inline templates/instructor_dashboard.html:40]: see above comment

File: templates/instructor_project.html [hunk 5]
```diff
@@ -45,7 +41,7 @@
     width: 80%;
     padding-left: 20px;
 }
-.nav{
+.nav-body{
```
Comment (bmyerz) [inline templates/instructor_project.html:44]: see above comment

File: templates/project_profile.html [hunk 6]
```diff
@@ -34,7 +30,7 @@
     width: 80%;
     padding-left: 20px;
 }
-.nav{
+.nav-body{
```
Comment (bmyerz) [inline templates/project_profile.html:33]: see above comment

File: templates/project.html (hunk 1 above)
Comment (zhouhuan200010pm) [reply templates/project.html:149]: That was a mistake, I think I miss that file when I change every document

**Pull request merged**